### ISSUE TITLE
Player's movement range should include all attackable locations.

### ISSUE THREAD

**Issue by james7132** [SmugilouGazilou/touhouemblem#18]
### Expected Behavior:
Player highlights playable character. The range of movement should highlight all movable squares in blue, and all attackable squares in red.

### Current Behavior:
Movement range UI only shows effective movement range, does not include total attack range.

**Comment by SmugilouGazilou** [comment]
Yeah. That too, is also a "Press ESC/X" thing. I should be implementing automatic showcasing soon though. That actually sounds really good.  But for now, you have to hit ESC on your player character.
